docs/UserGuide.adoc

```diff
@@ -72,6 +72,16 @@ Examples:
 Shows a list of all persons, along with their non-private details, in the address book. +
 Format: `list`
 
+== Sort existing list of persons by name: `sort name`
+
+Sorts the exisiting list of persons in the address book by name.
+Format: `sort name`
+
+== Sort existing list of persons by phone number: `sort phone`
+
+Sorts the existing list of persons in the address book by phone number.
+Format: `sort phone`
```

Comment:
Perhaps we can be clearer by saying "sort phone in increasing order" or something like that?
This is to differentiate 12345678 and 87654321.

Reply:
What about
> Sorts the entries by phone number in ascending order
? :)